src/autoscaler/db/sqldb/appmetric_sqldb_test.go

```diff
@@ -247,4 +248,78 @@ var _ = Describe("AppMetricSQLDB", func() {
 
 	})
 
+	Describe("PruneAppMetrics", func() {
```

Comment:
According to the code, when an error occurs, you will return it. I don't see how you guarantee that behavior.